.travis.yml

```diff
@@ -1,6 +1,16 @@
-sudo: false
```

Comment:
Why was this removed? Having it there makes Travis use their container based build system, which is faster then the regular one.